.Lib9c.Tests/Action/Sell4Test.cs

```diff
@@ -0,0 +1,234 @@
+namespace Lib9c.Tests.Action
+{
+    using System;
+    using System.Linq;
+    using Libplanet;
+    using Libplanet.Action;
+    using Libplanet.Assets;
+    using Libplanet.Crypto;
+    using Nekoyume;
+    using Nekoyume.Action;
+    using Nekoyume.Model;
+    using Nekoyume.Model.Item;
+    using Nekoyume.Model.State;
+    using Serilog;
+    using Xunit;
+    using Xunit.Abstractions;
+
+    public class Sell4Test
```

Comment:
How about making the naming of the action test class the same as the action?
e.g., SellTest contains unit tests of Sell[sell4]